README.md

```diff
@@ -148,49 +150,55 @@ Tidelift will coordinate the fix and disclosure.
+The currently released version is a stable 1.0.0 release. It is well-tested (97% coverage, ~500 test cases) and has most of the planned features already implemented. Plenty of companies and independent developers are using it in production with success.
 
-However, there are also plans for a lot more features like better TypeORM, Prisma and dataloader integration, custom decorators and metadata annotations support - [the full list of ideas](https://github.com/MichalLytek/type-graphql/issues?q=is%3Aissue+is%3Aopen+label%3A"Enhancement+%3Anew%3A") is available on the GitHub repo. You can also keep track of [development's progress on project board](https://github.com/MichalLytek/type-graphql/projects/1).
+However, there are also plans for a lot more features like better TypeORM, Prisma and DataLoader integration, custom decorators and metadata annotations support - [the full list of ideas](https://github.com/MichalLytek/type-graphql/issues?q=is%3Aissue+is%3Aopen+label%3A"Enhancement+%3Anew%3A") is available on the GitHub repo. You can also keep track of [development's progress on the project board](https://github.com/MichalLytek/type-graphql/projects/1).
 
 If you have any interesting feature requests, feel free to open an issue on GitHub so we can discuss that!
 
 ## Support
 
-TypeGraphQL is an MIT-licensed open source project. This framework is a result of the tremendous amount of work - sleepless nights, busy evenings and weekends.
+**TypeGraphQL** is an MIT-licensed open-source project. This framework is a result of the tremendous amount of work - sleepless nights, busy evenings and weekends.
 
-It doesn't have a large company that sits behind - its ongoing development is possible only thanks to the support by the community.
+It doesn't have a large company that sits behind it - its ongoing development is possible only thanks to the support of the community.
 
-[![](https://opencollective.com/typegraphql/donate/button.png?color=blue)](https://opencollective.com/typegraphql)
+[![donate](https://opencollective.com/typegraphql/donate/button.png?color=blue)](https://opencollective.com/typegraphql)
 
 ### Gold Sponsors 🏆
 
 > Please ask your company to support this open source project by [becoming a gold sponsor](https://opencollective.com/typegraphql/contribute/gold-sponsors-8340) and getting a premium technical support from our core contributors.
 
 ### Silver Sponsors 🥈
 
-| [<img src="https://raw.githubusercontent.com/MichalLytek/type-graphql/master/img/lovd.png" width="150">](https://www.lovd.com/) |
-| :-----------------------------------------------------------------------------------------------------------------------------: |
-|                                                [**Lovd**](https://www.lovd.com/)                                                |
+<!-- markdownlint-disable MD033 -->
+
+| [<img src="./images/lovd.png" width="150" alt="lovd" />](https://www.lovd.com) |
```

Comment:
Good to know that relative links now work on github readme :tada:

Reply:
@carlocorradini Now I remember why relative links in GitHub readme are wrong.
We publish this `Readme.md` to npm. The relative links does not work there.
Can we write a script for packaging the app that will replace the `./images` relative links with absolute urls to raw githubusercontent back? 😉